components/admin_console/data_grid/data_grid.tsx

```diff
@@ -52,8 +52,8 @@ type Props = {
     nextPage: () => void;
     previousPage: () => void;
 
-    search: (term: string) => void;
-    term: string;
+    search?: (term: string) => void;
```

Comment:
Small realization put would renaming this to `onSearch` require too many changes? Just feel it suites better but non-blocking cause I would think many other places would require the change.

Reply:
yea I agree, will make the change